nomad/csi_endpoint.go

```diff
@@ -357,7 +357,7 @@ func (v *CSIVolume) Register(args *structs.CSIVolumeRegisterRequest, reply *stru
 
 			*vol = *existingVol
 
-		} else if vol.Topologies == nil || len(vol.Topologies) == 0 {
+		} else if len(vol.Topologies) == 0 {
```

Comment:
huh, funny that this is equivalent to the nil check || length check. is this a new Go thing or has it always been like this?

Reply:
an empty slice has always been `==` to a nil slice in Go; now the linter just checks for it